API.Microsoft.Extensions.Hosting.Testing/WithDocs/Microsoft.Extensions.Hosting/FakeHostingExtensions.cs

```diff
@@ -0,0 +1,83 @@
+// Assembly 'Microsoft.Extensions.Hosting.Testing'
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Compliance.Testing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging.Testing;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Extension methods supporting host unit testing scenarios.
+/// </summary>
+[Experimental("EXTEXP0009", UrlFormat = "https://aka.ms/dotnet-extensions-warnings/{0}")]
+public static class FakeHostingExtensions
+{
+    /// <summary>
+    /// Starts and immediately stops the service.
+    /// </summary>
+    /// <param name="service">The tested service.</param>
+    /// <param name="cancellationToken">Cancellation token. See <see cref="T:System.Threading.CancellationToken" />.</param>
+    /// <returns>A <see cref="T:System.Threading.Tasks.Task" /> representing the asynchronous operation.</returns>
+    public static Task StartAndStopAsync(this IHostedService service, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
It's a bit misleading that an extension for an `IHostedService` is placed near the set of methods that operate on `IHost` and `IHostBuilder`

Reply:
At first it looked to me that the method is intended to start and stop an `IHost`

Reply:
Let's remove this function.